layouts/partials/components/footers/minimal.html

```diff
@@ -0,0 +1,28 @@
+<div class="row justify-content-center">
+    <div class="col-lg-2 col-md-2 col-sm-4 col-5 align-top mb-4 mb-md-0">
+        <img src="/images/Logo_base4NFDI_kurz_neg_rgb.png" alt="Your Image" class=" ">
+    </div>
+    <div class="col-lg-2 col-md-2 col-sm-4 col-5 mb-4 mb-md-0">
+        {{/* Display copyright license. */}}
+        {{ partial "site_footer_license" . }}
+    </div>
+    <div class="col-lg-2 col-md-2 col-sm-4 col-5 mb-4 mb-md-0">
+        {{ with site.Params.footer.mission }}
+        <p>
```

Comment:
I would suggest to center the text here and in the next paragraph, so all three text blocks have a uniform look

Reply:
On the div level we cannot change the text-align. It is on the paragraph level. But I can do so and make another pull request.